src/sentry/issues/occurrence_consumer.py

```diff
@@ -43,6 +52,92 @@ def create_ingest_occurences_consumer(
     )
 
 
+def save_event_from_occurrence(
+    data: Dict[str, Any],
+    **kwargs: Any,
+) -> Event:
+
+    from sentry.event_manager import EventManager
+
+    event_type = "platform"
+
+    with metrics.global_tags(event_type=event_type):
```

Comment:
What does this do?

Reply:
good q...just copied it from elsewhere. I need to do a final sweep before unmarking as draft to add metrics + error logging more fully